src/app/login/page.tsx

```diff
@@ -20,47 +20,26 @@ export default function LoginPage() {
           <button
             onClick={handleGoogleLogin}
             className="flex items-center justify-center gap-3 rounded-md bg-white px-4 py-2 text-sm font-medium text-gray-700 shadow transition hover:shadow-md dark:bg-gray-700 dark:text-gray-200"
           >
-            <svg viewBox="0 0 24 24" className="h-5 w-5">
-              <path
-                fill="#4285F4"
-                d="M22.56 12.25c0-.78-.07-1.53-.2-2.25H12v4.26h5.92c-.26 1.37-1.04 2.53-2.21 3.31v2.77h3.57c2.08-1.92 3.28-4.74 3.28-8.09z"
-              />
-              <path
-                fill="#34A853"
-                d="M12 23c2.97 0 5.46-.98 7.28-2.66l-3.57-2.77c-.98.66-2.23 1.06-3.71 1.06-2.86 0-5.29-1.93-6.16-4.53H2.18v2.84C3.99 20.53 7.7 23 12 23z"
-              />
-              <path
-                fill="#FBBC05"
-                d="M5.84 14.09c-.22-.66-.35-1.36-.35-2.09s.13-1.43.35-2.09V7.07H2.18C1.43 8.55 1 10.22 1 12s.43 3.45 1.18 4.93l2.85-2.22.81-.62z"
-              />
-              <path
-                fill="#EA4335"
-                d="M12 5.38c1.62 0 3.06.56 4.21 1.64l3.15-3.15C17.45 2.09 14.97 1 12 1 7.7 1 3.99 3.47 2.18 7.07l3.66 2.84c.87-2.6 3.3-4.53 6.16-4.53z"
-              />
-            </svg>
+            <img src="/google-logo.png" alt="Google" className="h-5 w-5" />
             <span>Googleでログイン</span>
-          </button>
+        </button>
           
-          <button
-            disabled
-            className="flex cursor-not-allowed items-center justify-center gap-3 rounded-md bg-white px-4 py-2 text-sm font-medium text-gray-400 shadow opacity-60 dark:bg-gray-700 dark:text-gray-500"
-          >
-            <svg viewBox="0 0 24 24" className="h-5 w-5 text-gray-400 dark:text-gray-500">
-              <path
-                fill="currentColor"
-                d="M12 2C6.477 2 2 6.477 2 12c0 4.42 2.865 8.166 6.839 9.489.5.092.682-.217.682-.482 0-.237-.008-.866-.013-1.7-2.782.603-3.369-1.34-3.369-1.34-.454-1.156-1.11-1.462-1.11-1.462-.908-.62.069-.608.069-.608 1.003.07 1.531 1.03 1.531 1.03.892 1.529 2.341 1.087 2.91.832.092-.647.35-1.088.636-1.338-2.22-.253-4.555-1.11-4.555-4.943 0-1.091.39-1.984 1.029-2.683-.103-.253-.446-1.27.098-2.647 0 0 .84-.268 2.75 1.026A9.578 9.578 0 0112 6.836c.85.004 1.705.114 2.504.336 1.909-1.294 2.747-1.026 2.747-1.026.546 1.377.202 2.394.1 2.647.64.699 1.028 1.592 1.028 2.683 0 3.842-2.339 4.687-4.566 4.935.359.309.678.919.678 1.852 0 1.336-.012 2.415-.012 2.743 0 .267.18.578.688.48C19.138 20.161 22 16.416 22 12c0-5.523-4.477-10-10-10z"
-              />
-            </svg>
-            <span>GitHubでログイン（準備中）</span>
-          </button>
+        <button
+          disabled
+          className="flex cursor-not-allowed items-center justify-center gap-3 rounded-md bg-white px-4 py-2 text-sm font-medium text-gray-400 shadow opacity-60 dark:bg-gray-700 dark:text-gray-500"
+        >
+          <img src="/github.png" alt="GitHub" className="h-5 w-5 text-gray-400 dark:text-gray-500" />
```

Comment:
[こちら](https://zenn.dev/dev8/articles/nextjs_image)を参考に修正してください！

```suggestion
          <Image src="/github.png" alt="GitHub" className="h-5 w-5 text-gray-400 dark:text-gray-500" />
```